src/pages/Map/components/MapLoadingBar/index.ts

```diff
@@ -0,0 +1 @@
+export { MapLoadingBar } from './MapLoadingBar';
```

Comment:
**suggestion** I think that it would be nice if you export `MapLoadingBarTestIds`

Reply:
good point :)